db/models/categories.models.js

```diff
@@ -1,6 +1,16 @@
 const connection = require('../connection')
+const fs = require("fs/promises");
+
+exports.fetchEndpoints = () => {
+    return fs.readFile('endpoints.json', 'utf-8').then((result) => {
+        const jsObject = JSON.parse(result)
+        return jsObject
+    })
+}
```

Comment:
There should not be a model for this endpoint as you are not querying the database. You can add this in the controller to just read the file.